Floofbot/Modules/Utilities.cs

```diff
@@ -139,5 +141,60 @@ public async Task ServerInfo()
             await Context.Channel.SendMessageAsync("", false, embed.Build());
 
         }
+        [Command("serverlist")]
+        [Summary("Returns a list of servers that the bot is in")]
+        [RequireOwner]
+        public async Task ServerList()
+        {
+            List<SocketGuild> guilds = new List<SocketGuild>(Context.Client.Guilds);
+            List<PaginatedMessage.Page> pages = new List<PaginatedMessage.Page>();
+
+            foreach (SocketGuild g in guilds)
+            {
+                List<EmbedFieldBuilder> fields = new List<EmbedFieldBuilder>();
+
+                fields.Add(new EmbedFieldBuilder()
+                {
+                    Name = $"Owner",
+                    Value = $"{g.Owner.Username}#{g.Owner.Discriminator} | ``{g.Owner.Id}``",
+                    IsInline = false
+                });
+                fields.Add(new EmbedFieldBuilder()
+                {
+                    Name = $"Server ID",
+                    Value = g.Id,
+                    IsInline = false
+                });
+                fields.Add(new EmbedFieldBuilder()
+                {
+                    Name = $"Members",
+                    Value = g.MemberCount,
+                    IsInline = false
+                });
+
+                pages.Add(new PaginatedMessage.Page
+                {
+                    Author = new EmbedAuthorBuilder { Name = g.Name },
+                    Fields = new List<EmbedFieldBuilder>(fields),
+                    ThumbnailUrl = (Uri.IsWellFormedUriString(g.IconUrl, UriKind.Absolute) ? g.IconUrl : null)
+                });
+            }
+            var pager = new PaginatedMessage
+            {
+                Pages = pages,
+                Color = Color.DarkGreen,
+                Content = "Here are a list of servers that I am in!",
+                FooterOverride = null,
+                Options = PaginatedAppearanceOptions.Default,
+                TimeStamp = DateTimeOffset.UtcNow
+            };
+            await PagedReplyAsync(pager, new ReactionList
+            {
+                Forward = true,
+                Backward = true,
+                Jump = true,
+                Trash = true
+            }, true);
```

Comment:
What's this true parameter for?

Reply:
Only the person who ran the command will have their reactions counted